src/server/users/controllers.js

```diff
@@ -2,7 +2,7 @@ import { KeyUtils, ENTITY, STATE } from '../key-utils';
 import { S3Connector } from '../aws-s3';
 import db from '../database';
 import { APIError } from '../ErrorHandler';
-import { CreatedPostManager, SavedPostManager } from './models';
+import { SavedPostManager, CreatedPostManager } from './models';
```

Comment:
Ok I just wondered. I think it is better not to change it unless it has meanings